ome/omero-gallery
# Babel build

This was more painful than expected!
Adds a simple build step using babel to compile newer JS to support IE and older browsers. Also adds polyfills for features missing from IE11.

To test:
 - Everything should work on IE11 and possibly older IE (not tested).

To build:
```
    npm install
    npm run build
```
This will watch the ```src``` directory and compile any changes into the gallery static directory.
NB: Both the src and the built files are committed so we don't need a build step in the install process (as we did with OMERO.figure for a while). 
This will all change when we switch to a library/framework after first release.
cc @jburel
